### PR TITLE
Patch overlooked strings from Python un-namespacing

### PR DESCRIPTION
**Pull request by soltanmm-google**

### CODE REVIEW

**Comment by soltanmm-google** [comment]
https://grpc-testing.appspot.com/view/Artifacts/job/gRPC_build_artifacts/2365/

**Comment by soltanmm-google** [comment]
If interop tests appear to work nicely, then yay, if they don't, then back to the drawing board.

**Comment by soltanmm-google** [comment]
Well, instead of a segfault there's a timeout. That's... almost progress. Still can't get `v1.0.x` to fail to build locally in the first place.

**Comment by soltanmm-google** [comment]
Running this branch manually on the interop test machine doesn't show any irregularities, and *building* shouldn't have any race conditions (heck as much as I'd wanted to make the build parallel distutils doesn't allow that). This is weird. Really, really weird.

```
atash@grpc-canary-interop3:~/grpc$ time ./tools/run_tests/run_interop_tests.py -l python -s python --use_docker
Seen --use_docker flag, will run interop tests under docker.

IMPORTANT: The changes you are testing need to be locally committed
because only the committed changes in the current branch will be
copied to the docker environment.
START: Building interop docker images.
PASSED: build_docker_python [time=340.8sec; retries=0:0]
SUCCESS: All docker images built successfully.
START: interop_server_python
PASSED: cloud_to_cloud:python:python_server:large_unary [time=2.6sec; retries=0:0]
PASSED: cloud_to_cloud:python:python_server:empty_unary [time=2.6sec; retries=0:0]
PASSED: cloud_to_cloud:python:python_server:ping_pong [time=2.6sec; retries=0:0]
PASSED: cloud_to_cloud:python:python_server:empty_stream [time=2.6sec; retries=0:0]
PASSED: cloud_to_cloud:python:python_server:client_streaming [time=2.6sec; retries=0:0]
PASSED: cloud_to_cloud:python:python_server:server_streaming [time=2.6sec; retries=0:0]
PASSED: cloud_to_cloud:python:python_server:cancel_after_begin [time=2.9sec; retries=0:0]
PASSED: cloud_to_cloud:python:python_server:timeout_on_sleeping_server [time=3.4sec; retries=0:0]
PASSED: cloud_to_cloud:python:python_server:cancel_after_first_response [time=3.4sec; retries=0:0]
SUCCESS: All tests passed
Removing docker image grpc_interop_python:4b11cf42-6723-4f58-8b7c-cd9f659b1700

real	5m52.178s
user	0m0.684s
sys	0m0.596s
atash@grpc-canary-interop3:~/grpc$ git branch
  jenkins-docker
  master
* praytell-babbage-if-you-do-the-wrong-thing-will-it-still-do-the-right-thing
  v1.0.x
```

**Comment by soltanmm-google** [comment]
Might fix #9108.

File: src/python/grpcio_health_checking/MANIFEST.in [hunk 1]
```diff
@@ -1,4 +1,4 @@
 include grpc_version.py
 include health_commands.py
-graft grpc
```
Comment (nathanielmanistaatgoogle) [inline src/python/grpcio_health_checking/MANIFEST.in]: I don't understand how the change on this line relates to the rest of the pull request. Have I missed something in the commit messages?

File: tools/distrib/python/grpcio_tools/MANIFEST.in [hunk 2]
```diff
@@ -2,6 +2,6 @@ include grpc_version.py
 include protoc_deps.py
 include protoc_lib_deps.py
 include README.rst
-graft grpc
```
Comment (nathanielmanistaatgoogle) [inline tools/distrib/python/grpcio_tools/MANIFEST.in]: Here too.

File: src/python/grpcio_health_checking/MANIFEST.in (hunk 1 above)
Comment (soltanmm-google) [inline src/python/grpcio_health_checking/MANIFEST.in]: It's changed for the same reason as everything else: things used to be in a package called `grpc`, and are now in a package called `grpc_$SOMETHING`.

**Comment by dgquintas** [comment]
Please document failures. If no issue exists, create one and assign it. If one exists, ping it and/or assign it.

**Comment by soltanmm-google** [comment]
Done.

**Pull request merged**